netinterface.py

```diff
@@ -348,3 +374,21 @@ def loadFromNetDb(jdata, hwaddr):
 
         nic.addIPv6(modev6, ipv6addr, gatewayv6)
         return nic
+
+class NetInterfaceV6(NetInterface):
```

Comment:
The design underlying this class ought to be presented somewhere.  Since there is no design doc in this project, a good place could be the commit message.